src/index.html

```diff
@@ -1,13 +1,407 @@
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--tablet-1-2 grid__item--desktop-1-4">
+                  <div class="place__photo-container">
+                    <div class="place__photo place__photo--5">
+                    </div>
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--tablet-3-6 grid__item--desktop-5-12">
+                  <div class="place__photo-container">
+                    <div class="place__photo place__photo--6">
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="place__description">
+              Audio sunglasses
+            </div>
+          </div>
+
+          <div class="grid grid--tablet grid--desktop">
+            <div class="grid__item grid__item--tablet-2-5 grid__item--desktop-5-8">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="page__section buy" id="how-to-buy">
+        <div class="container">
+          <div class="grid grid--tablet grid--desktop">
+            <div class="
+              grid__item
+              grid__item--tablet-1-3
+              grid__item--desktop-1-5
+            ">
+              <h2 class="buy__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+            <div class="
+              grid__item
+              grid__item--tablet-4-6
+              grid__item--desktop-6-10
+            ">
+              <h3 class="buy__subtitle">
+                Free 2-day shipping and returns
+              </h3>
+              <h3 class="buy__subtitle">
+                90-day risk-free trial
+              </h3>
+              <h3 class="buy__subtitle">
+                World class customer service
+              </h3>
+              <h3 class="buy__subtitle">
+                My Bose account management
+              </h3>
+              <p class="buy__paragraph">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+
+            </div>
+          </div>
+        </div>
+      </section>
+
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--tablet grid--desktop">
+            <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <form action="./">
```

Comment:
Add the id to your form to make your scripts work
```suggestion
              <form action="./" id="form">
```